Makefile

```diff
@@ -0,0 +1,96 @@
+# opentrons modules makefile
+# https://github.com/Opentrons/opentrons-modules
+
+SHELL := /bin/bash
+
+CI_SYSTEM ?= $(shell uname)
+
+LIBRARIES_DIR := libraries
+MODULES_DIR := modules
+INO_DIR := $(HOME)/arduino_ide
+
+ARDUINO_VERSION ?= 1.8.5
+ARDUINO_SAMD_VER ?= 1.6.20
+ADAFRUIT_SAMD_VER ?= 1.2.9
+OPENTRONS_BOARDS_VER ?= 1.2.0
+
+ADAFRUIT_BOARD_URL := https://adafruit.github.io/arduino-board-index/package_adafruit_index.json
+OPENTRONS_BOARD_URL := https://s3.us-east-2.amazonaws.com/opentrons-modules/package_opentrons_index.json
+
+ifeq ($(CI_SYSTEM), Darwin)
+	ARDUINO := $(INO_DIR)/Arduino.app/Contents/MacOS/Arduino
+	ARDUINO_ARCHIVE := arduino-$(ARDUINO_VERSION)-macosx.zip
+	UNPACK_CMD := unzip $(ARDUINO_ARCHIVE) -d $(INO_DIR)
+	ARDUINO_SKETCHBOOK_LOC := $(HOME)/Documents/Arduino
+	ARDUINO15_LOC := $(HOME)/Library/Arduino15
+else
+	ARDUINO := $(INO_DIR)/arduino
+	ARDUINO_ARCHIVE := arduino-$(ARDUINO_VERSION)-linux64.tar.xz
+	UNPACK_CMD := tar xf $(ARDUINO_ARCHIVE) -C $(INO_DIR)/ --strip-components=1
+	ARDUINO_SKETCHBOOK_LOC := $(HOME)/Arduino
+	ARDUINO15_LOC := $(HOME)/.arduino15
+endif
+
+BUILDS_DIR := $(HOME)/.arduino_builds
```

Comment:
[nitpick] If at all possible let's try and keep this in the tree, like `./build` or something